src/main/java/net/minecraftforge/client/IForgeArmPose.java

```diff
@@ -35,9 +35,10 @@
      * This method should be used to apply all wanted transformations to the player when the ArmPose is active.
      * You can use {@link LivingEntity#getTicksUsingItem()} and {@link LivingEntity#getUseItemRemainingTicks()} for moving animations.
      *
-     * @param model  The humanoid model
-     * @param entity The humanoid entity
+     * @param model   The humanoid model
+     * @param entity  The humanoid entity
+     * @param isRight True if the animation is applied to the right arm, false if to the left arm.
```

Comment:
Instead of a boolean, it might be better to use `net.minecraft.world.entity.HumanoidArm.LEFT` / `RIGHT`. That is meaningful by itself, without needing to check the parameter name / docs.